lib/vrl/stdlib/src/encode_key_value.rs

```diff
@@ -310,10 +308,10 @@ mod tests {
             ],
             want: Err(format!(r"invalid field value type at index 1: {}",
                     value::Error::Expected {
-                        got: Kind::Integer,
-                        expected: Kind::Bytes
+                        got: Kind::integer(),
+                        expected: Kind::bytes()
```

Comment:
This change was needed because the `value` crate no longer uses the enum('ish) structure from `bitflags`. All of these were updated mechanically.